packages/tldraw/src/components/TopPanel/PageMenu/PageMenu.tsx

```diff
@@ -66,7 +66,7 @@ function PageMenuContent({ onClose }: { onClose: () => void }) {
   const currentPageId = app.useStore(currentPageIdSelector)
 
   const handleCreatePage = React.useCallback(() => {
-    app.createPage(undefined, intl.formatMessage({ id: 'new.page' }))
```

Comment:
Let's keep this as it is. There's no guarantee that a user would want their new page to be related to their current page.

Reply:
well we could consider checking if the name has a number in it and if yes it will increase it otherwise it will make a page named "new page"?

Reply:
also considering that the user is gonna have to rename the "new page" anyway, it might be as well just follow the previous page numbering, worst case scenario they are gonna delete it, like they would have done anyway

Reply:
alright cool